api/SubmitJob/src/main.py

```diff
@@ -0,0 +1,27 @@
+import json
+from os import getenv
+
+import boto3
+
+client: boto3 = boto3.client('batch')
+
+
+def lambda_handler(event, context):
+    parameters = json.loads(event['body'])
+    job = client.submit_job(jobName=parameters['granule'],
+                            jobQueue=getenv('JobQueue'),
+                            jobDefinition=getenv('JobDefinition'),
+                            parameters={
+                                'granule': parameters['granule']
+                            }
+                            )
+
+    response = {
+        'Job Name': job['jobName'],
+        'Job Id': job['jobId']
```

Comment:
Not a fan of spaces in dictionary keys.  Can this just be `response=job` to pass along exactly the `submit_job` response? (we'll need to tweak it with more content later, yes)

Reply:
The response from batch as a bunch of stuff nobody will care about, it looks like 

```json
{
    "ResponseMetadata": {
        "RequestId": "a435ea3c-d405-4be4-95e5-f18721659387",
        "HTTPStatusCode": 200,
        "HTTPHeaders": {
            "date": "Thu, 14 May 2020 21:25:25 GMT",
            "content-type": "application/json",
            "content-length": "64",
            "connection": "keep-alive",
            "x-amzn-requestid": "a435ea3c-d405-4be4-95e5-f18721659387",
            "x-amz-apigw-id": "MimS6FsoPHcFvsQ=",
            "x-amzn-trace-id": "Root=1-5ebdb745-68db30466536b804ec191c3a;Sampled=0"
        },
        "RetryAttempts": 0
    },
    "jobName": "foo",
    "jobId": "a6481a34-8ef7-4529-90b9-19efa0685e0e"
}
```

Reply:
think I prefer a minimal response, then:
```
{
  "jobId": job['jobId']
}
```